src/lib/y2users/linux/writer.rb

```diff
@@ -124,10 +124,10 @@ def write
       # @return [Y2User::Config]
       attr_reader :initial_config
 
-      # The list of issues generated while writting changes to the system
+      # The list of issues generated while writting changes to the system. See #write
       #
       # @return [Y2Issues::List]
-      attr_accessor :issues
+      attr_reader :issues
```

Comment:
I am still not sure why it is needed. Do you have example of usage? I kind of expect something like
```ruby
issues = Linux::Writer.write
if !issues.empty?
  ... reporting
end
```

Reply:
It's a private `attr_reader` just for _documenting_ it and to avoid using `@` all the time (now `issues << Y2Issues::Issue.new(...)` is used in two different methods.

I can drop it if it hurts :)

Reply:
Well, what I mean that I would not use `@issues` at all and just pass it in given methods. This way you do not hold issues from previous runs.

Reply:
Ah, I see what you meant now